docs/reference/cloud/elastic-cloud-kibana-settings.md

```diff
@@ -63,6 +63,9 @@ stack: ga 9.2
 :    A list of allowed email recipient patterns (`to`, `cc`, or `bcc`) that can be used with email connectors. If you attempt to send an email to a recipient that does not match the allowed patterns, the action will fail. The failure message indicates that the email is not allowed.
 
 ### Version 9.1+ [ec_version_9_1]
+```{applies_to}
+stack: ga 9.1
+```
```

Comment:
Is there a problem with including prose (in the title) that mentions the specific version alongside a badge that will say `Planned` for some period of time? This is something I struggled with in a few places.

Reply:
this page needs to be probably removed ultimately, so I think we can just live with the discomfort for now

Reply:
Yup we've had that in a few places and for now I used to "quick fix" route, which is than instead of reorganizing the entire page into tabs or so, I at least add a badge under the title that will indicate that that version is "planned" for the time being. I've had a few instances with 9.2 already too.